src/ArmTemplates/Commands/Executors/ExtractorExecutor.cs

```diff
@@ -401,6 +427,7 @@ async Task GenerateSingleAPIWithRevisionsTemplates()
             var globalServicePolicyTemplate = await this.GeneratePolicyTemplateAsync(baseFilesGenerationDirectory);
             var productApiTemplate = await this.GenerateProductApisTemplateAsync(singleApiName, multipleApiNames, baseFilesGenerationDirectory);
             var productTemplate = await this.GenerateProductsTemplateAsync(singleApiName, multipleApiNames, baseFilesGenerationDirectory, apiTemplateResources);
+            await this.GenerateGroupsTemplateAsync(baseFilesGenerationDirectory);
```

Comment:
Shouldn't we do something with the outcome?

Reply:
I do not know - do we need to include this outcome to the `masterTemplate` or not. Due to this task is a user's request (with users implementation), I think we can leave it as it is for now. If they want groupsTemplate to exist in `masterTemplate`, we can add it in future any time.

Reply:
Well, shouldn't we have a consistent experience for things? I'd say yes.